modules/thermo-cycler/thermo-cycler-arduino/lid.cpp

```diff
@@ -241,19 +247,17 @@ bool Lid::open_cover()
   }
   motor_on();
   bool res;
-  if(move_angle(LID_OPEN_SWITCH_PROBE_ANGLE))  // move down to reach bottom switch
+  // move down a bit to release latch
+  unsigned long push_steps = abs(LID_OPEN_DOWN_MOTION_ANGLE) * STEPS_PER_ANGLE;
```

Comment:
Why do this manually instead of `move_angle`?

Reply:
Because move angle exits when a switch press is detected. With the optical switch, we need to move down a bit more while the switch is 'ON' to let the latch actually lock.

Reply:
Changed `move_angle` a bit so that it can be used here